### DISCUSSION TITLE
Renovate doesn't recreate closed PRs for major releases after a new patch/minor version is released

### DISCUSSION THREAD

**Discussion by danports** [renovatebot/renovate#19990] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

I sometimes run into the following issue with the default Renovate configuration:

1. Some dependency releases a new major version (say, 7.0.0), Renovate opens a PR, and I merge it, because I do whatever Renovate tells me to do. 😉 
2. That new major version has a bug (as new major versions do), so I revert the Renovate PR merge, open an issue with the upstream project, work through it with them, and eventually a new patch or minor release (say, 7.0.1) gets released.
3. Renovate does nothing, because a closed PR already exists for the 7.x major upgrade ("Update dependency Blah to v7").

Ideally, I'd like Renovate to open a new PR in this case for the 7.0.1 release. I see there's a `recreateClosed` configuration option, but that seems heavy-handed - I usually don't want to recreate closed PRs.

Any thoughts on the best approach here?

**Comment by rarkins** [comment]
Related issue: #16820

Best thing you can do is use Dependency Dashboard and approve the blocked PR
